packages/application/src/shell.ts

```diff
@@ -253,20 +253,11 @@ export class LabShell extends Widget implements JupyterFrontEnd.IShell {
     this.layout = rootLayout;
 
     // Connect change listeners.
-    this._tracker.currentChanged.connect(
-      this._onCurrentChanged,
-      this
-    );
-    this._tracker.activeChanged.connect(
```

Comment:
This change alone is worth the price of admission.

Reply:
Yes, when I saw that I was exceedingly happy.